pymc/gp/gp.py

```diff
@@ -866,6 +870,16 @@ def conditional(self, name, Xnew, pred_noise=False, given=None, jitter=0.0, **kw
         return pm.MvNormal(name, mu=mu, cov=cov, **kwargs)
 
 
+@conditioned_vars(["X", "Xu", "y", "sigma"])
+class MarginalSparse(MarginalApprox):
+    def __init__(self, approx="VFE", *, mean_func=Zero(), cov_func=Constant(0.0)):
+        warnings.warn(
+            "gp.MarginalSparse has been renamed to gp.MarginalApprox.",
+            FutureWarning,
+        )
```

Comment:
@twiecki I have this FutureWarning here, is that what you mean by add deprecation warning?

Reply:
Yes, exactly.